common/src/main/java/eu/europeana/fulltext/repository/AnnoPageRepository.java

```diff
@@ -389,16 +389,15 @@ private List<String> getDcTypes(List<AnnotationType> annoTypes) {
                 .collect(Collectors.toUnmodifiableList());
     }
 
-
-    public List<AnnoPage> getAnnoPages(String dsId, String lcId) {
+    public List<AnnoPage> getAnnoPages(String datasetId, String localId, String pageId, String lang, boolean includeDeprecated) {
```

Comment:
Best to make this two separate methods:

- One that returns a single AnnoPage, querying on `dsId`, `lcId`, `pgId` and `lang`;
- The other to return a list of AnnoPages, querying on `dsId`, `lcId` and `pgId`.

That way it's clear what each field combination returns.

Ideally, the method names should also indicate that full AnnoPage objects aren't returned – just projection of certain fields.